internal/pkg/cli/storage_init.go

```diff
@@ -0,0 +1,235 @@
+// Copyright Amazon.com, Inc or its affiliates. All rights reserved.
+
+package cli
+
+import (
+	"fmt"
+
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/config"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/color"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/log"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/workspace"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+const (
+	dynamoDBStorageType = "Dynamo DB"
+	s3StorageType       = "S3 Bucket"
```

Comment:
Maybe:
```suggestion
	dynamoDBStorageType = "DynamoDB" 
	s3StorageType       = "S3"
```
So that it's either ("DynamoDB table" and "S3 bucket") or ("DynamoDB" and "S3")

Reply:
Or 
```suggestion
	dynamoDBStorageType = "DynamoDB (NoSQL database)"
	s3StorageType       = "S3 (Object store)"
```